tools/test-case-generators/format-request-map.json

```diff
@@ -494,11 +525,7 @@
           }
         ],
         "modules": [],
-        "groups": [
```

Comment:
@thozza @achilleas-k I had to empty this group for this target as it added extra packages that were not part of the v1 for Fedora. It turns out actually, that the qcow2 already incorporates the `@core` package as part of it's package list, so no harm done here. Let me know if you want me to put it back and I'll do so :smile: 

This also raises the question, I wonder, that adding the same group package through the blueprint somehow incorporates extra rpms.

Reply:
Yes, this is a known issue with the way we depsolve the two sets of packages (base and blueprint).  We're working on fixing it but it's a pretty big change.
In short, the blueprint packages and depsolved separately without restrictions (no excluded packages), whereas in the base package set we exclude some weak dependencies.  So if `@core` is added to the blueprint, it pulls in all those optional packages as well.
See https://github.com/osbuild/osbuild-composer/issues/2386

Reply:
got it. I'll remove it for now to match the rpms from old fedora manifests, but please let me know if you'd like it back.